### ISSUE TITLE
Clock on task bar is vanished.

### ISSUE THREAD

**Issue by tk03125** [valinet/ExplorerPatcher#3248]
I'm using Windows 11 Pro Version 23H2 OS build 22631.3527 with ExplorerPatcher version 22621.3527.65.3.
Clock on task bar is vanished.
If uninstall ExplorerPatcher then Clock appear.

**Comment by pyrates999** [comment]
Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by tk03125** [comment]
Thank you for your reply.
I have installed ExplorerPatcher for another environment then this issue was NOT recreated.
I have rechecked on my environment and I found following unexpected change.
Open properties and task bar.
Click “Customize system icons in the notification area” then System Icons of “Clock” is showed “Off”.

I have changed above to “On” then problem is solved.

This chage is unexpected but I cannot recreate this issue then I have closed this issue.
Thank you.
